src/pages/index.tsx

```diff
@@ -26,8 +26,8 @@ const HomePage = () => (
     <Hero />
     <Features />
     <Grid>
-      <Box as="hr" marginBottom={0} />
-      <Box paddingVertical={80} paddingVerticalMd={120}>
+      <Box as="hr" margin={0} />
```

Comment:
Does `hr` has some margin by default?

Reply:
Yes. When I've deleted margin={0} it has margin top and bottom by default.

Reply:
Got it, didn't remember that. Thanks for refreshing!